components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/user/impl/UserInfoJSONResponseBuilder.java

```diff
@@ -194,6 +194,8 @@ public String getResponseString(OAuth2TokenValidationResponseDTO tokenResponse)
 
         if (StringUtils.isNotEmpty(cacheEntry.getEssentialClaims())) {
             lstEssential = getEssentialClaims(cacheEntry.getEssentialClaims());
+        } else {
+            lstEssential = new ArrayList<>();
```

Comment:
Can we have a meaningful name for this variable?
The current variable name 'lstEssential' does not reflect its purpose at least to me 😕